schema/schema.cc

```diff
@@ -906,8 +917,15 @@ std::ostream& schema::describe(replica::database& db, std::ostream& os, bool wit
     os << "\n    AND memtable_flush_period_in_ms = " << memtable_flush_period();
     os << "\n    AND min_index_interval = " << min_index_interval();
     os << "\n    AND read_repair_chance = " << read_repair_chance();
-    os << "\n    AND speculative_retry = '" << speculative_retry().to_sstring() << "';";
-    os << "\n";
+    os << "\n    AND speculative_retry = '" << speculative_retry().to_sstring() << "'";
```

Comment:
> Add `speculative_retry`, `paxos_grace_seconds`, `cdc` and `synchronous_updates`
options to schema description.

It looks like `speculative_retry` was already described.